keyboards/jones/v03/matrix.c

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
Could this be converted to matrix lite? 
https://docs.qmk.fm/#/custom_matrix?id=lite

Reply:
I changed that to "matrix lite" on commit [c164fc5](https://github.com/qmk/qmk_firmware/pull/11130/commits/c164fc57c769b77a85374da3221ddce030283766).

Should I change more?

Reply:
Sorry Missed that somehow